src/olympia/migrations/922-drop-auto-validation-stuff.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE `file_validation`
+    DROP COLUMN `signing_trivials`,
+    DROP COLUMN `signing_lows`, 
+    DROP COLUMN `signing_mediums`,
+    DROP COLUMN `signing_highs`,
+    DROP COLUMN `passed_auto_validation`;
+DROP TABLE `validation_annotations`;
```

Comment:
The columns removed would need to be nullable for that. That's doable but I don't think it matters much, since it's only going to break uploads with the old code while we push.